Header header
actionlib_msgs/GoalID goal_id

uint8 GOAL = 1
uint8 PATH = 2
uint8 EXPLORE = 3
uint8 type

uint8[] goal
